parodos-model-api/src/main/java/com/redhat/parodos/workflow/parameter/WorkFlowValueProvider.java

```diff
@@ -0,0 +1,9 @@
+package com.redhat.parodos.workflow.parameter;
+
+import java.util.List;
+
+public interface WorkFlowValueProvider {
```

Comment:
Shouldn't it be _WorkFlowParameterValueProvider_? But also if we'd like to keep it generic for workflow and workflow task parameter, it might be better to name it as _WorkParameterValueProvider_